### PR TITLE
feat: Add Velas Blockchain Currency & NPM Package

### PR DESCRIPTION
**Pull request by Z4karia**
### 📝 Description

- Added VLX Cryptocurrency
- Added NPM Package for Velas App Integration

### ❓ Context

- **Impacted projects**: `hw-app-velas` 
- **Linked resource(s)**: `https://github.com/LedgerHQ/ledgerjs/pull/827` 

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Comment by Z4karia** [comment]
The PR is ready @n4l5u0r

File: libs/ledgerjs/packages/hw-app-velas/package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        "LedgerWallet",
+        "Velas",
+        "VLX",
+        "NanoS",
+        "Blue",
+        "Hardware Wallet"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/LedgerHQ/ledgerjs"
```
Comment (n4l5u0r) [inline libs/ledgerjs/packages/hw-app-velas/package.json]: pas le bon repo

File: libs/ledgerjs/packages/hw-app-velas/package.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    },
+    "bugs": {
+        "url": "https://github.com/LedgerHQ/ledgerjs"
```
Comment (n4l5u0r) [inline libs/ledgerjs/packages/hw-app-velas/package.json]: same here

File: libs/ledgerjs/packages/hw-app-velas/package.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    },
+    "homepage": "https://github.com/LedgerHQ/ledgerjs",
```
Comment (n4l5u0r) [inline libs/ledgerjs/packages/hw-app-velas/package.json]: same here

**Review by n4l5u0r — CHANGES_REQUESTED** [review]

File: libs/ledgerjs/packages/hw-app-velas/README.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
+<img src="https://user-images.githubusercontent.com/211411/34776833-6f1ef4da-f618-11e7-8b13-f0697901d6a8.png" height="100" />
+
+[Github](https://github.com/LedgerHQ/ledgerjs/),
```
Comment (n4l5u0r) [inline libs/ledgerjs/packages/hw-app-velas/README.md]: not correct github repo

**Review by n4l5u0r — CHANGES_REQUESTED** [review]

**Review by n4l5u0r — APPROVED** [review]

**Pull request merged**